src/coreComponents/physicsSolvers/NonlinearSolverParameters.cpp

```diff
@@ -157,6 +162,10 @@ void NonlinearSolverParameters::postProcessInput()
   {
     GEOS_ERROR( " timeStepIncreaseIterLimit should be smaller than timeStepDecreaseIterLimit!!" );
```

Comment:
I know it's not part of this PR, but can we take the opportunity to remove the double exclamation mark here. I find it frightening.